content/millennium/r4/care-provision/nutrition-order.md

```diff
@@ -111,6 +123,25 @@ List an individual NutritionOrder by its id:
 
 <%= disclaimer %>
 
+#### Patient Authorization Request
+
+    GET https://fhir-ehr.cerner.com/4/ec2458f2-1e24-41c8-b71b-0e701af7583d/NutritionOrder/328575687
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_nutrition_order_patient_access) %>
```

Comment:
reuse R4_NUTRITION_ORDER_PATIENT_ACCESS_ENTRY instead of r4_nutrition_order_patient_access

No need of that i guess.